src/components/help/help.test.js

```diff
@@ -219,4 +219,128 @@ context("Tests for Help component", () => {
       }
     );
   });
+
+  describe("Accessibility tests for Help component", () => {
```

Comment:
for all tests below we are not passing the parameter to the `title` like we did in previous tests, like:

`should render Help with %s as a parameter, (parameter)` because the `%s` is missing and all titles for each run look the same